pipelines/templates/bosh-lite-cf-edge.yml

```diff
@@ -123,7 +123,7 @@ jobs:
             BOSH_TARGET: <%= full_deployment_name +  '.' + domain_name %>
             BOSH_LITE_NAME: {{deployment-name}}
             RUBYGEM_MIRROR: {{rubygem-mirror}}
-            DOMAIN_NAME: {{domain_name}}
+            DOMAIN_NAME: {{domain-name}}
```

Comment:
In fact, both are available !
 - `domain-name` from concourse credentials (through public-credentials or may be overridden with lastpass)
 - `domain_name` from erb, (only in ruby templating), but using value from concourse credentials using `full_config["domain-name"]` mechanism

If you prefer, I can rename `domain-name`, and `domain_name`, to `domain` or to any other name without _ nor -.